lib/commands/command-env.bash

```diff
@@ -15,7 +16,10 @@ shim_env_command() {
   fi
 
   shim_env() {
-    "$env_cmd" "${env_args[@]}"
+    # Yes, the syntax for the expansion is ugly and confusing, but it is what we
+    # need to do to make it work with the nounset option. See
+    # https://stackoverflow.com/q/7577052/ for the details
+    "$env_cmd" ${env_args[@]+"${env_args[@]}"}
```

Comment:
In an effort to capture minimum versions we support, I would like capture the min Bash version that supports this in the docs if we could